gtdynamics/dynamics/Chain.cpp

```diff
@@ -0,0 +1,102 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2021, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file Chain.cpp
+ * @brief Create a serial kinematic chain.
+ * @author Dan Barladeanu, Frank Dellaert.
+ */
+
+#include <gtdynamics/dynamics/Chain.h>
+
+namespace gtdynamics {
+
+Chain operator*(const Chain &chainA, const Chain &chainB) {
+  // expect 6 rows in jacobians
+  if (chainA.axes().rows() != 6 || chainB.axes().rows() != 6) {
```

Comment:
Let’s catch that in constructor rather than here.

Reply:
done